recovery/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="kinrecovery_restore">Restaurar</string>
+    <string name="kinrecovery_backup">Backup</string>
+
+    <!--Backup-->
+    <string name="kinrecovery_keep_your_kin_safe">Mantenha seu Kin seguro</string>
+    <string name="kinrecovery_backup_description">Fazendo o backup do seu saldo atual irá lhe permitir\n recuperá-lo, caso você troque o telefone, apague o app\n ou seus dados.</string>
+    <string name="kinrecovery_let_s_go">Vamos lá</string>
+    <string name="kinrecovery_create_password">Criar uma Senha</string>
+    <string name="kinrecovery_create_password_description">Para fazer o backup do seu Kin.\n primeiro crie uma senha forte. Ela deve ter:\n (9 caracters, 1 maiúscula, 1 caracter especial (#, !, @) e 1 número)</string>
```

Comment:
it seems like after `\n` you have space is it intended?

Reply:
no, updated.